index.html

```diff
@@ -17,28 +17,26 @@ <h1>Binge Pad</h1>
     </header>
     <!-- background -->
     <div class="no-signal">
-    
-   <!-- button now within container of background styling -->
-    <button id="new-list" class="tv">FUNKY TV IMAGE OR GIF!</button>
+    </div>
+  
+   <!-- button now outside container of background styling -->
+   <button aria-label="Add a new list" id="new-list" class="tv">
+     <img src="tv-gif.gif" aria-hidden="true" alt="Gif of plasticine TV with dancing antennas"/>
+     <p>Add New List</p>
+   </button>
 
     <main>
       <div class="main"></div>
     </main>
-   
-      <button aria-label="Add a new list" id="new-list" class="tv">
-        <img src="tv-gif.gif" aria-hidden="true" alt="Gif of plasticine TV with dancing antennas"/>
-        <p>Add New List</p>
-      </button>
 
-    </div>
 
 
     <!-- static class attribute 
     <div class="tv-static"></div>
     -->
-    <script src="test-helpers.js"></script>
+   <!-- <script src="test-helpers.js"></script> -->
```

Comment:
Helpful that it's there for use later when we are all ready to create the tests 😸 🐰